ossfuzz/compress_fuzzer.cc

```diff
@@ -0,0 +1,22 @@
+#include <stddef.h>
+#include <stdint.h>
+#include <stdlib.h>
+#include "lz4.h"
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
+{
+  size_t const compressed_dest_size = LZ4_compressBound(size);
+  char *const dest_buffer = (char *)malloc(compressed_dest_size);
```

Comment:
nit: note that any invocation of `malloc()` can fail.
Therefore, it should be tested for success (`dest_buffer != NULL`).

Reply:
Agreed, will fix up.